plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1500,17 +1545,25 @@ def update_auth_policy_server(self, authentication_policy_server):
 
             # Authentication and Policy Server exists, check update is required
             # Edit API not working, remove this
-            self.format_payload_for_update(self.have.get("authenticationPolicyServer")[auth_server_index].get("details"),
-                                           self.want.get("authenticationPolicyServer")[auth_server_index]).check_return_status()
-            is_ise_server_enabled = self.have.get("authenticationPolicyServer")[auth_server_index].get("details").get("isIseEnabled")
-            if not (is_ise_server_enabled or self.requires_update(self.have.get("authenticationPolicyServer")[auth_server_index].get("details"),
-                                                                  self.want.get("authenticationPolicyServer")[auth_server_index],
-                                                                  self.authentication_policy_server_obj_params)):
+            have_auth_server_details = self.have.get("authenticationPolicyServer")[auth_server_index].get("details")
+            want_auth_server_details = self.want.get("authenticationPolicyServer")[auth_server_index]
+
+            self.format_payload_for_update(have_auth_server_details, want_auth_server_details).check_return_status()
+
+            is_ise_server_enabled = have_auth_server_details.get("isIseEnabled")
+            ise_server_requires_update = False
+            if is_ise_server_enabled:
+                ise_server_requires_update = self.check_ise_server_updation_status(have_auth_server_details,
+                                                                                   want_auth_server_details)
```

Comment:
if ise_server_requires_update:
        self.log("Cisco ISE server requires an update based on configuration changes.")
    else:
        self.log("Cisco ISE server does not require any updates.")